etc/lfrc.example

```diff
@@ -35,9 +35,9 @@ map O $mimeopen --ask $f
 # This command is called when current file is not a directory. You may want to
 # use either file extensions and/or mime types here. Below uses an editor for
 # text files and a file opener for the rest.
-cmd open ${{
+cmd open &{{
     case $(file --mime-type -Lb $f) in
-        text/*) $EDITOR $fx;;
+        text/*) lf -remote "send $id \$$EDITOR \$fx";;
         *) for f in $fx; do setsid $OPENER $f > /dev/null 2> /dev/null & done;;
```

Comment:
This is optional (I can also do it at some point), but could you surround all the shell variables with quotes, so that it doesn't choke if files have spaces in them? See https://www.shellcheck.net/wiki/SC2086 for more details.

Reply:
This file sets ifs to "\n", so variables won't be split on spaces. $fx should not be quoted because we need to split it into separate file arguments

Reply:
That makes sense. Thanks, @p-ouellette. Consider this resolved (I don't have the ability to actually resolve the conversation).